### PR TITLE
Update projects.md

### PR DESCRIPTION
**Pull request by XXXWeii**
添加博客案例

### CODE REVIEW

**Comment by XXXWeii** [comment]
Mr.Hope看看我的成果哈哈哈哈哈

File: docs/theme/src/zh/demo/projects.md [hunk 1]
```diff
@@ -221,6 +221,13 @@ blog:
     repo: https://github.com/Mister-Hope/Mister-Hope.github.io
     preview: /assets/image/mrhope.jpg
 
+- name: XXXWeii
```
Comment (Mister-Hope) [inline docs/theme/src/zh/demo/projects.md:224]: Wrong indent
```suggestion
  - name: XXXWeii 
```

**Review by Mister-Hope — CHANGES_REQUESTED** [review]

**Comment by Mister-Hope** [comment]
Also why do you think you should put you things before others?

File: docs/theme/src/zh/demo/projects.md [hunk 2]
```diff
@@ -221,6 +221,13 @@ blog:
     repo: https://github.com/Mister-Hope/Mister-Hope.github.io
     preview: /assets/image/mrhope.jpg
 
+  - name: XXXWeii 
+   desc: 带着思念期待着下次相见 
+   logo: https://xxxweii.github.io/XXXWeiiBlog/logo.jpg
+   url: https://xxxweii.github.io/XXXWeiiBlog 
+   repo: https://github.com/XXXWeii/XXXWeiiBlog 
+   preview: https://xxxweii.github.io/XXXWeiiBlog/assets/images/XXXWeii.png
```
Comment (Mister-Hope) [inline docs/theme/src/zh/demo/projects.md:229]:
```suggestion
   desc: 带着思念期待着下次相见 
    logo: https://xxxweii.github.io/XXXWeiiBlog/logo.jpg
    url: https://xxxweii.github.io/XXXWeiiBlog 
    repo: https://github.com/XXXWeii/XXXWeiiBlog 
    preview: https://xxxweii.github.io/XXXWeiiBlog/assets/images/XXXWeii.png 
```

File: docs/theme/src/zh/demo/projects.md [hunk 3]
```diff
@@ -221,6 +221,13 @@ blog:
     repo: https://github.com/Mister-Hope/Mister-Hope.github.io
     preview: /assets/image/mrhope.jpg
 
+  - name: XXXWeii 
+   desc: 带着思念期待着下次相见
```
Comment (Mister-Hope) [inline docs/theme/src/zh/demo/projects.md:225]:
```suggestion
    desc: 带着思念期待着下次相见 
```

**Pull request closed without merging**